akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -892,6 +892,10 @@ backup node.
   A shared LevelDB instance is a single point of failure and should therefore only be used for testing
   purposes. Highly-available, replicated journal are available as `Community plugins`_.
 
+.. note::
+
+  This plugin has been supplanted by :ref:`Journal Proxy <journal-proxy>`.
```

Comment:
We should also start a migration guide migration-guide-2.4.x-2.5.x.rst, use https://github.com/akka/akka/blob/master/akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst as template. Nothing fancy, just mentioning what you wrote here in the note.